pages/docs/tutorials/app-example-config.mdx

```diff
@@ -0,0 +1,672 @@
+
+    $ grouparoo generate sqlite:app product_db
+
+    🦘 Grouparoo: generate sqlite:app product_db
+
+    ✅ wrote /private/tmp/my-sync-project/config/apps/product_db.js
+
+Take a look at the output. Grouparoo generated a config file for you in `config/apps/product_db.js`! Before we look at that, let's use [the `validate` command](/docs/cli/config#validate) to see if everything is working right:
+
+    $ grouparoo validate
+
+    🦘 Grouparoo: validate
+
+    Validating 1 objects...
+    notice: [ app ] testing app product_db (product_db) threw error: Could not find SQLite database: /path/to/app-config-example/...
+    warning: [ config ] error with app `product_db` (product_db): Error: error testing app product_db (product_db) - Could not find SQLite database: /path/to/app-config-example/...
+
+    ❌ Validation failed - 1 validation error
+
+Well, shoot! It didn't work. But that's actually a good thing – it means our validator is working. Because we haven't told Grouparoo where our database is.
+
+Open up that config file. The only thing you should have to adjust is [the `file` option](https://github.com/grouparoo/app-example-config/blob/a7eda5b16fe8ab15e9dc5cf49f389d43bfba9a19/config/apps/product_db.js#L9). This should be the path to your database. It can be absolute or relative to the root of your Grouparoo project.
+
+```js
+exports.default = async function buildConfig() {
+  return [
+    {
+      class: "app",
+      id: "product_db",
+      name: "product_db",
+      type: "sqlite",
+      options: {
+        // Change this!
+        file: "/path/to/sqlite/database.sqlite",
+      },
+    },
+  ];
+};
+```
+
+If you want it to match the example project exactly, first move the `sqlite-source.sqlite` file you downloaded into a `data` directory within your project, and then rename it to `source.sqlite`. Otherwise, you can determine the path to the database and add it directly.
+
+### Validate and Apply Config Changes
+
+Keep using the `validate` command until you have a working connection. Eventually you should see it working as expected.
+
+    $ grouparoo validate
+
+    🦘 Grouparoo: validate
+
+    Validating 1 objects...
+    info: [ config ] validated App `product_db` (product_db)
+
+    ✅ Validation succeeded - 1 config objects OK!
+
+Hooray! When you're ready let's [apply those changes](/docs/cli/config#apply), which means we're saving them to the database.
+
+    $ grouparoo apply
+
+    🦘 Grouparoo: apply
+
+    Applying 1 objects...
+    notice: [ config ] created App `product_db` (product_db)
+
+    ✅ Config applied - 1 config objects up-to-date!
+
+Now we have an app and we can add some Sources!
+
+## Step 5: Add Users Source and Properties
+
+Next, it's time to add our Sources and their Properties. A Source is the place your customer data lives. In this case, we're mocking a local data warehouse type of setup. Properties are attributes for your Profiles (in Grouparoo, we refer to users as _Profiles_).
```

Comment:
No idea if "local datawarehouse" is a good metaphor or confusing to folks that already have one.